scotty

```diff
@@ -294,6 +300,7 @@ def run(args):
 
     if '--rov' in args:
         print_color("Running ROV main loop:\n")
+        print("python %s" % get_file_path("rov/rov.py"))
         os.system("python %s" % get_file_path("rov/rov.py"))
```

Comment:
I don't remember if there is a reason for using `os.system` here, but `run_commands([python %s" % get_file_path("rov/rov.py")` is what's used elsewhere to print and call the command. Should maybe switch to that to be consistent.